test/e2e-api/admin/posts.test.js

```diff
@@ -418,6 +418,27 @@ describe('Posts API', function () {
         res2.body.posts[0].status.should.eql('draft');
     });
 
+    it('Can\'t change the newsletter_id of a post', async function () {
```

Comment:
I don't think this test is doing what it says - it's returning a 422 because the newsletter_id 123 doesn't exist, not because updating the newsletter_id is blocked.

Reply:
It does return a 422 because the property is blocked in the API as it isn't defined in the admin schema. I've changed to a valid id using the newsletters test fixtures.